apps/frontend/src/components/products/styles.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from "@emotion/styled";
+
+const ProductsHeader = styled.div`
+  display: flex;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  align-items: center;
+
+  h2,
+  h4 {
+    margin: 0;
+  }
+
+  margin-bottom: 1rem;
+`;
+
+const ProductsHeaderOptions = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+  align-items: center;
+  margin-bottom: 0.5rem;
+
+  button {
+    background-color: #eeffef;
```

Comment:
You could have used some styles template file to store all the colors, font sizes and so on. That would ensure the components are consistent across different pages